tests/unit/lms/validation/_api_test.py

```diff
@@ -146,7 +146,9 @@ def all_fields(self):
 @pytest.fixture
 def json_request(pyramid_request):
     def _make_json_request(data, exclude=None):
-        pyramid_request.content_type = "application/json"
+        pyramid_request.content_type = pyramid_request.headers[
+            "content-type"
+        ] = "application/json"
```

Comment:
`pyramid_request.headers["content-type"]` is the full header string with params, e.g. `"application/json; charset=utf-8"`. `pyramid_request.content_type` is the content type only, e.g. `"application/json"`. We need to set both because our won code reads `pyramid_request.content_type` whereas webargs reads `pyramid_request.headers["content-type"]`. It's not important for the test that `pyramid_request.headers["content-type"]` actually include the `; charset=utf-8"` (or any params), just `"application/json"` is enough.